### PR TITLE
Authentificationsecurise

### PR DESCRIPTION
**Pull request by ThomasDCW**
I add authentification on the frond end form, when you have a good email and a good pass present on the db, you get the 200 response in console and a tostify notification.
When you have a wrong email/pass you have 401 response and a tostify notification/
Next step we should have the way to redirect the user on the app

### CODE REVIEW

**Review by Dash293 — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: admin/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+VITE_BACKEND_URL=http://localhost:5000
+/admin
```
Comment (LoicBrassart) [inline admin/.env:2]: :scream: 
WHY IS THERE A .ENV FILE HERE ?!
@LuigiMenez ! SOMEONE!!! ALAAAAARM !!!!!!! :skull: :warning:

File: frontend/src/components/SignIn/index.jsx [hunk 2]
```diff
@@ -1,31 +1,53 @@
...
+import axios from "axios";
+import { toast } from "react-toastify";
 import SSignIn from "./style";
 
 export default function SignIn() {
+  const [formData, setFormData] = useState({ email: "", password: "" });
+  const hSubmit = (evt) => {
+    evt.preventDefault();
+    axios
+      .post("http://localhost:5000/auth/login", formData)
```
Comment (LoicBrassart) [inline frontend/src/components/SignIn/index.jsx:12]: Don't use "http://localhost:5000" anymoe, use env variables

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
ALAAAAAAAAAAAAAARRRRRMMMMMMMMMM !!!!!!!!!!

**Review by Dash293 — APPROVED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Okk for me.

File: frontend/src/components/SignIn/index.jsx [hunk 3]
```diff
@@ -1,31 +1,55 @@
...
 import SSignIn from "./style";
 
 export default function SignIn() {
+  const [formData, setFormData] = useState({ email: "", password: "" });
+  const hSubmit = (evt) => {
+    evt.preventDefault();
+    axios
+      .post(`${import.meta.env.VITE_BACKEND_URL}${"/auth/login"}`, formData)
+      .then(() => {
+        toast("Wow so easy!");
```
Comment (LoicBrassart) [inline frontend/src/components/SignIn/index.jsx:14]: Customize this message

File: frontend/src/components/SignIn/index.jsx [hunk 4]
```diff
@@ -1,31 +1,55 @@
...
+      })
+      .catch(() => {
+        toast.error("Aww, it broke!");
```
Comment (LoicBrassart) [inline frontend/src/components/SignIn/index.jsx:17]: Same

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
When asked to improve a bit of code, you should not implement *new features*: if you miss something, the whole PR is held back

**Review by ilalande — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

File: frontend/src/main.jsx [hunk 5]
```diff
@@ -12,6 +12,7 @@ root.render(
   <React.StrictMode>
     <Reset />
     <BrowserRouter>
+      <ToastContainer />
```
Comment (LuigiMenez) [inline frontend/src/main.jsx:15]: Why 2 toastContainer?

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Two toastContainer?

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**